apps/voting_web/web/elm/ChatApp/Views.elm

```diff
@@ -30,15 +30,46 @@ submitNominationView model =
 nominationsListView : Model -> Html Msg
 nominationsListView model =
     defaultLayout (p [] [ connectionMessage model.connectionMessage
-         ,div [] (List.map nominationView model.nominations)
+         ,div [] (List.map (nominationView model) model.nominations)
          ])
 
-nominationView : Nomination -> Html Msg
-nominationView nomination =
-    div []
-        [ span [] [ text nomination.nominee ]
-        , span [] [ text (" - " ++ nomination.nominatorName) ]
-        ]
+nominationView : Model -> Nomination -> Html Msg
+nominationView model nomination =
+    let
+        event = chooseEvent model
+    in
+        form [ Events.onSubmit event ]
```

Comment:
Oh yeah nice. I initially had the `chooseEvent` logic in the let but didn't finish cleaning up.